continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -2,80 +2,102 @@
 {% set new_patch = major_minor_patch[2] | int + 1 %}
 {% set version = (major_minor_patch[:2] + [new_patch]) | join('.') + environ.get('VERSION_SUFFIX', '') %}
 {% set dask_version = environ.get('DASK_CORE_VERSION', '0.0.0.dev') %}
+{% set cython_enabled = build_ext == "cython" %}
```

Comment:
Jinja unfortunately doesn't affect selectors. So we would need to perform this comparison in selectors

Reply:
Ah thanks for the clarification here - good to know for the future 🙂

Reply:
Ofc. Showed another way we could do this in this review ( https://github.com/dask/distributed/pull/5831#pullrequestreview-890316547 ). Though ok with either approach 🙂